bundle/bundle.go

```diff
@@ -10,14 +12,52 @@ import (
 
 var globs = []string{"*.plugin.zsh", "*.zsh", "*.sh", "*.zsh-theme"}
 
-// Bundle is a git-based bundle/plugin
+// BundleType is an interface for different types of bundles
+type BundleType interface {
```

Comment:
I think calling this `Type` would be good enough.

BTW, since this is not being used by other packages, maybe we can let this private?